web/templates/compare.html

```diff
@@ -65,7 +65,7 @@ <h5 class="text-center">{{ lang2_friendlyname|linebreaksbr }}'s Implementation</
                 </div>
                 <div class="card">
                     <div class="card-body">
-                        <a href="https://github.com/codethesaurus/codethesaur.us/blob/main/web/thesauruses/{{ lang2 }}/{{ concept }}.json">
+                        <a href="https://github.com/codethesaurus/codethesaur.us/edit/main/web/thesauruses/{{ lang2 }}/{{ concept }}.json">
```

Comment:
This works, but will also need to change line 61 to do the same thing. (On the compare page, the left language just opens the file, the right one edits. They don't match!)

Reply:
okay , will work on this one